### PR TITLE
show invalid fields on UBO form mount

### PR DESCRIPTION
**Pull request by Epimodev**
On the onboarding, when there is an invalid UBO and open it, we don't where the error come from.  
This PR adds auto validation on UBO form only when we edit it. So the user will be able to see directly what are invalid fields

### CODE REVIEW

**Review by bloodyowl — APPROVED** [review]

File: packages/shared-business/src/components/BeneficiaryForm.tsx [hunk 1]
```diff
@@ -384,6 +384,28 @@ export const BeneficiaryForm = forwardRef<BeneficiaryFormRef | undefined, Props>
 
     const hasBeenSubmittedOnce = useRef(false);
 
+    useEffect(() => {
+      if (initialState != null) {
+        // validate all fields on mount to display errors on edition open
+        Promise.all([
+          validateField("firstName"),
```
Comment (zoontek) [inline packages/shared-business/src/components/BeneficiaryForm.tsx:391]: Maybe I should add a `validateForm` to ux-form, WDYT?

Comment (zoontek) [inline packages/shared-business/src/components/BeneficiaryForm.tsx:391]: I think you can call `submitForm` instead and noop the result.

Comment (Epimodev) [reply packages/shared-business/src/components/BeneficiaryForm.tsx:391]: yes I didn't have this idea, will try

Comment (Epimodev) [reply packages/shared-business/src/components/BeneficiaryForm.tsx]: It works perfectly well thanks for this top. Just pushed 449eceb9a89367528a5e2375459a2bda2d6a1dad

**Review by zoontek — APPROVED** [review]

**Pull request merged**